src/components/internals/localized-input-toggle/localized-input-toggle.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import FlatButton from '@commercetools-uikit/flat-button';
+import { WorldIcon } from '@commercetools-uikit/icons';
+import messages from '../messages/localized-input/messages';
+
+const LocalizedInputToggle = (props) => {
+  const intl = useIntl();
+
+  const labelMessage = props.isOpen ? props.hideMessage : props.showMessage;
```

Comment:
why not inline this into L15?

Reply:
For readability, and because it's used both on L14 and L15.